packages/frontend/src/pages/Space.tsx

```diff
@@ -1,29 +1,23 @@
-import {
-    Intent,
-    Menu,
-    NonIdealState,
-    PopoverPosition,
-} from '@blueprintjs/core';
-import { MenuItem2, Popover2 } from '@blueprintjs/popover2';
+import { Intent, NonIdealState } from '@blueprintjs/core';
```

Comment:
hey @IrakliJani , wanted to ask about this import should we change it or it's fine ?

Reply:
@jayanand05 not in the scope of the ticket so it's ok, we'll have other tickets to hunt down remaining blueprint imports at some point